src/main/java/Bern.java

```diff
@@ -0,0 +1,59 @@
+import bern.functionalities.Parser;
+import bern.functionalities.Storage;
+import bern.functionalities.TaskList;
+import bern.functionalities.Ui;
+import bern.model.Task;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+/**
+ * This class is where the main method resides.
+ */
+public class Bern {
+
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+    private Parser parser;
+
+    /**
+     * Constructor for class Bern.
+     */
+    public Bern() {
+        ui = new Ui();
+        storage = new Storage();
+        taskList = new TaskList();
+        parser = new Parser();
+    }
+
+    /**
+     * Method that runs the programme from an instance of Bern.
+     */
+    public void run() {
+        Scanner myObj = new Scanner(System.in);
+
+        ArrayList<Task> arListTask = new ArrayList<>();
```

Comment:
Like that you used an arraylist to store the tasks instead of an array due to arraylist being easily resizable. Perhaps you can call it arListTasks instead of arListTask to follow the naming convention.